tests/steps/deploy/k8s/test_crd.py

```diff
@@ -0,0 +1,38 @@
+from pathlib import Path
+
+import pytest
+
+from mpyl.project import load_project
+from mpyl.steps import Input
+from mpyl.steps.deploy.k8s.chart import ChartBuilder
+from mpyl.steps.deploy.k8s.resources.crd import to_yaml
+from tests import root_test_path
+from tests.test_resources import test_data
+
+
+def get_builder_for_resource(resource_path: Path) -> ChartBuilder:
+    step_input = Input(load_project(resource_path, Path('project.yml'), True), test_data.RUN_PROPERTIES, None)
+    return ChartBuilder(step_input)
+
+
+class TestCrdChart:
```

Comment:
Just a note to check later: should be also add scenario's for the other chart types?